packages/playground/src/dashboard/components/user_nodes.vue

```diff
@@ -197,21 +192,13 @@ export default {
     const uptime = ref();
     const nodesCount = ref();
 
-    onMounted(async () => {
-      const nodes = await getUserNodes();
-      if (nodes) {
-        await Promise.all(
-          nodes.map(async (n, _) => {
-            n.uptime = +(await getNodeUptimePercentage(n.nodeId));
-            return n.uptime;
-          }),
-        );
-      }
-    });
+    onMounted(async () => await getUserNodes());
```

Comment:
You can pass the function directly to the onMounted hock instead of calling it inside an arrow function.

```suggestion
    onMounted(getUserNodes);
```

Reply:
thanks